docs/data/material/getting-started/example-projects/example-projects.md

```diff
@@ -4,7 +4,7 @@
 
 ## Official examples
 
-The following starter projects are all available in the MUI Core [`/examples`](https://github.com/mui/material-ui/tree/master/examples) folder.
+The following starter projects are all available in the Material UI [`/examples`](https://github.com/mui/material-ui/tree/master/examples) folder.
```

Comment:
Yup, same as in https://github.com/mui/mui-x/commit/2f26cdd2e4aec0a4f2e811425f98a30b2eaa4e05. I think we should teach developer MUI System, Base UI, Joy UI, and Material UI rather than a soup of the 4.

Reply:
Yes, I'm somewhat confident that we should go towards not using the "MUI Core" terminology that much anymore. 😬